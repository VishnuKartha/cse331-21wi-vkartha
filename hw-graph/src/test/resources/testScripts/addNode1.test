# Creates a single graph and adds one node to it

# Create a Graph
CreateGraph graph1

# Add a node
AddNode graph1 n1
