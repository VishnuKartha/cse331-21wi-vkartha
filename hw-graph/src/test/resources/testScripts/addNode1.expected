# Creates a single graph and adds one node to it

# Create a Graph
created graph graph1

# Add a Node
added node n1 to graph1
